as/cNFT/assembly/models/persistent_nft_contract_metadata.ts

```diff
@@ -0,0 +1,66 @@
+import { storage } from 'near-sdk-as'
+
+const NFT_SPEC = 'nft-1.0.0'
+const NFT_NAME = 'Nft'
+const NFT_SYMBOL = 'NFT'
+
+@nearBindgen
+export class NFTContractMetadata {
+    spec: string
+    name: string
+    symbol: string
+    icon: string
+    base_uri: string
+    reference: string
+    reference_hash: string
+    packages_script: string
+    render_script: string
+    style_css: string
+    parameters: string
+}
+
+export function defaultNFTContractMetadata(): NFTContractMetadata {
+    return {
+        spec: NFT_SPEC,
+        name: NFT_NAME,
+        symbol: NFT_SYMBOL,
+        icon: '',
+        base_uri: '',
+        reference: '',
+        reference_hash: '',
+        packages_script: '',
+        render_script: '',
+        style_css: '',
+        parameters: '',
+    }
+}
+
+@nearBindgen
+export class PersistentNFTContractMetadata {
```

Comment:
Why not call this `PersistentContractMetadata` like `PersistentTokenMetadata`?
The same for `NFTContractMetadata` and the file name

Reply:
This is naming used by `NEAR` team: https://nomicon.io/Standards/NonFungibleToken/Metadata.html
Agree it's not perfect tho!

Reply:
@achrafismyname @AshenI99